### ISSUE TITLE
Full Accessibility & 508 Office Audit VAOS Community Care Provider Selection

### ISSUE THREAD

**Issue by laurenernest** [department-of-veterans-affairs/va.gov-team#23731]
## Steps to complete the Full Accessibility & 508 Audit: 
- [x] VFS: Verify all accessibility defects that were identified at Staging Review have been fixed before requesting a full audit *(VSP understands that other teams may be accountable for fixing the defects identified at Staging Review. These defects are exempt and will not block a team from scheduling this touchpoint)* #19542
- [x] VFS Product Manager: Create this issue and fill in your team and feature name in the title
- [x] VFS: Link to this issue once created in [#vfs-platform-support](https://dsva.slack.com/archives/CBU0KDSB1) in Slack; tag @ Trevor Pierce
- [x] VSP: Create sub-issues attached to this epic with findings. Audits are performed on a first-come-first-serve basis and are scheduled with each biweekly sprint
- [x] VFS: Coordinate with the VA 508 Office to address accessibility findings
- [x] VSP: Update [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=1116695886)

## Artifacts
- Access information for the tool or feature in staging. Do not put staging credentials in your `va.gov-team` ticket; store or reference them in a .md file in the `va.gov-team-sensitive` repository
  - Use [Judy](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/staging-test-accounts-vaos.md)
  - Ensure a residential address is in the user's Profile before proceeding the audit
- Link to verification that accessibility checks are complete:
  - #18708
- List of known issues (with links to tickets when applicable)

## Additional information
As part of this review, VSP will send your product to VA's 508 office for approval (as applicable).

Please refer to the [Collaboration Cycle README](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/README.md) on Github for more information about the Collaboration Cycle.

**Comment by noahgelman** [comment]
@laurenernest @shiragoodman 

No accessibility issues

**Comment by laurenernest** [comment]
Great! That audit turnaround was quick too! Closing ticket.

**Comment by 1Copenut** [comment]
@laurenernest Much appreciated. Let us know if you need contact creds for the VA 508 office or have questions about submitting the feature for their final Quick Test.

**Comment by 1Copenut** [comment]
@laurenernest All items requested by the VA 508 office added to this epic. I'll respond in the email thread and let them know. Thank you!

**Comment by laurenernest** [comment]
The 3 issues have been resolved. Closing epic.

**Comment by 1Copenut** [comment]
Thank you @laurenernest Would you mind letting Linsey at VA 508 know these issues have been completed and ready for a second look?

**Comment by laurenernest** [comment]
Sent! Got an OOO notification through 6/15 so it'll be a few days before we hear back.

**Comment by laurenernest** [comment]
Got validation from Linsey that the defects have been fixed.
